.jenkins/pytorch/test.sh

```diff
@@ -613,7 +613,4 @@ else
   test_custom_backend
   test_torch_function_benchmark
   test_benchmarks
-  if [[ "${BUILD_ENVIRONMENT}" == *linux-xenial-py3.6-gcc7-test* || "${BUILD_ENVIRONMENT}" == *linux-xenial-py3.6-gcc5.4-test* ]]; then
-    test_python_gloo_with_tls
```

Comment:
Hmm, should we perhaps moove gloo test to some supported config?

Reply:
Probably. Right now it's not running anyway - no 3.6 workflows.

Reply:
I've added an issue https://github.com/pytorch/pytorch/issues/78398